src/fabric/web/index.html

```diff
@@ -42,3 +42,4 @@
     <img src="[%Researchpixel%]" class="creative__pixel">
     <img src="[%Viewabilitypixel%]" class="creative__pixel">
 </div>
+[%[%thirdPartyJSTracking%]%]
```

Comment:
Looks like you’ve got a little extra, there!

Reply:
> Looks like you’ve got a little extra, there!

Ah yes missed that, thank you. Updated and re committed now.